{"uid":"ccac4bf55799c8d9","name":"Oops...we found an error preparing this test file: cypress/e2e/api/openweathermap/weatherSearch.cy.js","fullName":"Oops...we found an error preparing this test file: cypress/e2e/api/openweathermap/weatherSearch.cy.js","time":{"start":1680358252630,"stop":1680358252630,"duration":0},"status":"broken","statusMessage":"Oops...we found an error preparing this test file:\n\n  > cypress/e2e/api/openweathermap/weatherSearch.cy.js\n\n","statusTrace":"\n\nError: Webpack Compilation Error\n./cypress/e2e/api/openweathermap/weatherSearch.cy.js\nModule build failed (from /home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/babel-loader/lib/index.js):\nSyntaxError: /home/runner/work/cypress-intermediario-v2/cypress-intermediario-v2/cypress/e2e/api/openweathermap/weatherSearch.cy.js: Unexpected token, expected \",\" (133:53)\n\n  131 |     }).then(response => {\n  132 |       expect(response.status).to.equal(200)\n> 133 |       expect(response.body.sys.country).to.equal(\"{/\"type/\":2,/\"id/\":2033898,/\"country/\":/\"BR/\",/\"sunrise/\":1680340471,/\"sunset/\":1680383169}/\")\n      |                                                      ^\n  134 |       expect(response.body.name).to.equal(\"SÃ£o Paulo\")\n  135 |     })\n  136 |   })\n    at handle (/home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/@packages/server/node_modules/@cypress/webpack-preprocessor/dist/index.js:231:23)\n    at finalCallback (/home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:257:39)\n    at /home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:306:14\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at AsyncSeriesHook.lazyCompileHook (/home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/tapable/lib/Hook.js:154:20)\n    at /home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:304:22\n    at Compiler.emitRecords (/home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:499:39)\n    at /home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:298:10\n    at /home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:485:14\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at AsyncSeriesHook.lazyCompileHook (/home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/tapable/lib/Hook.js:154:20)\n    at /home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:482:27\n    at /home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/neo-async/async.js:2818:7\n    at done (/home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/neo-async/async.js:3522:9)\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at AsyncSeriesHook.lazyCompileHook (/home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/tapable/lib/Hook.js:154:20)\n    at /home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:464:33\n    at /home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/@packages/server/node_modules/graceful-fs/graceful-fs.js:143:16\n    at /home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/@packages/server/node_modules/graceful-fs/graceful-fs.js:143:16\n    at /home/runner/.cache/Cypress/12.0.2/Cypress/resources/app/node_modules/@packages/server/node_modules/graceful-fs/graceful-fs.js:61:14\n    at FSReqCallback.oncomplete (node:fs:188:23)\n\nThis occurred while Cypress was compiling and bundling your test code. This is usually caused by:\n\n- A missing file or dependency\n- A syntax error in the file or one of its dependencies\n\nFix the error in your code and re-run your tests.","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"suite","value":"cypress/e2e/api/openweathermap/weatherSearch.cy.js"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"ccac4bf55799c8d9.json","parameterValues":[]}